module/VuFind/src/VuFind/ILS/Driver/Virtua.php

```diff
@@ -444,11 +444,9 @@ public function getHolding($id, ?array $patron = null, array $options = [])
             }
 
             // Call number
-            if ($row['ITEM_CALL_NUM'] != null) {
-                $call_num = $row['ITEM_CALL_NUM'];
-            } else {
-                $call_num = $row['BIB_CALL_NUM'];
-            }
+            $call_num = $row['ITEM_CALL_NUM'] != null
+                ? $row['ITEM_CALL_NUM']
+                : $row['BIB_CALL_NUM'];
```

Comment:
This can be further simplified:

```suggestion
            $call_num = $row['ITEM_CALL_NUM'] ?? $row['BIB_CALL_NUM'];
```

It wasn't automatically refactored because the comparison was `!=` rather than `!==`, but I believe this should be safe nonetheless.